Cabal/src/Distribution/Simple/Program/GHC.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE BlockArguments #-}
```

Comment:
`BlockArguments` is not supported in 8.4.4, which is one of our mandatory supported compilers for lib:Cabal